platform/rct/hal/usart_stm32/usart_device_stm32.c

```diff
@@ -66,27 +68,27 @@ static int initQueues()
     int success = 1;
 
     /* Create the queues used to hold Rx and Tx characters. */
-    xUsart0Rx = xQueueCreate(UART_QUEUE_LENGTH,
+    xUsart0Rx = xQueueCreate(UART_RX_QUEUE_LENGTH,
                              (unsigned portBASE_TYPE)sizeof(signed portCHAR));
-    xUsart0Tx = xQueueCreate(UART_QUEUE_LENGTH + 1,
+    xUsart0Tx = xQueueCreate(UART_TX_QUEUE_LENGTH + 1,
```

Comment:
Fair enough, Wasn't sure if there was a reason for them, didn't want to pee in the pool. (Though in the future, hopefully we can keep cross-driver contamination out of the mix entirely, I only touched this because I was out of heap space)